rita_client/src/dashboard/mod.rs

```diff
@@ -62,160 +64,156 @@ use rita_common::network_endpoints::*;
 
 pub fn start_client_dashboard(rita_dashboard_password: u16) {
```

Comment:
this variable should be called 'rita_dashboard_port' not password. This isn't your doing just noticed during review.

Reply:
fixed